virttest/libvirt_xml/devices/tpm.py

```diff
@@ -0,0 +1,64 @@
+"""
+tpm device support class(es)
+
+http://libvirt.org/formatdomain.html#elementsTpm
+"""
+
+from virttest.libvirt_xml import accessors
+from virttest.libvirt_xml.devices import base
+
+
+class Tpm(base.UntypedDeviceBase):
+
+    __slots__ = ('tpm_model', 'backend')
+
+    def __init__(self, virsh_instance=base.base.virsh):
+        accessors.XMLAttribute('tpm_model', self,
+                               parent_xpath='/',
+                               tag_name='tpm',
+                               attribute='model')
+        accessors.XMLElementNest('backend', self, parent_xpath='/',
+                                 tag_name='backend', subclass=self.Backend,
+                                 subclass_dargs={
+                                     'virsh_instance': virsh_instance})
+        super(Tpm, self).__init__(
+            device_tag='tpm', virsh_instance=virsh_instance)
+        self.xml = '<tpm/>'
```

Comment:
this line is not necessary, the class UntypedDeviceBase will do it

Reply:
Done. Thx.